script-generator/src/main/java/com/coolbitx/wallet/signing/scriptlib/EtcScript.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.coolbitx.wallet.signing.scriptlib;
+
+import com.coolbitx.wallet.signing.utils.HexUtil;
+import com.coolbitx.wallet.signing.utils.ScriptArgumentComposer;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler;
+import com.coolbitx.wallet.signing.utils.ScriptData;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler.HashType;
+import com.coolbitx.wallet.signing.utils.ScriptAssembler.SignType;
+import com.coolbitx.wallet.signing.utils.ScriptData.Buffer;
+
+public class EtcScript {
+
+    public static void listAll() {
+        System.out.println("Etc: \n" + getCROScript() + "\n");
+    }
+
+    /*
+    E7
+    2A
+    85 09C74AFE1F
+    82 5208
+    94 A3255ECFE3F6727A62D938F4C29B2F73C361B26C
+    83 989680
+    80
+    03
+    80
+    80
+     */
+    public static String getCROScript() {
+        ScriptArgumentComposer sac = new ScriptArgumentComposer();
+        ScriptData argTo = sac.getArgument(20);
+        ScriptData argValue = sac.getArgumentRightJustified(10);
+        ScriptData argGasPrice = sac.getArgumentRightJustified(10);
+        ScriptData argGasLimit = sac.getArgumentRightJustified(10);
+        ScriptData argNonce = sac.getArgumentRightJustified(8);
+
+        String script = new ScriptAssembler()
+                // set coinType to 3C
+                .setCoinType(0x3D)
+                .arrayPointer()
+                // nonce
+                .rlpString(argNonce)
+                // gasPrice
+                .rlpString(argGasPrice)
+                // gasLimit
+                .rlpString(argGasLimit)
+                // toAddress
+                .copyString("94").copyArgument(argTo)
+                // value
+                .rlpString(argValue)
+                // data
+                .copyString("80")
+                // chainId v
+                .copyString("61", Buffer.CACHE1)
```

Comment:
chain id should be 41

Reply:
It's 61 for ETC according to https://chainlist.org/